### PR TITLE
fix:add build context

### PR DESCRIPTION
**Pull request by kakaZhou719**
### Describe what this PR does / why we need it
1, add context path at local build
2, delete -c flag on build cmd

### Describe how to verify it
test report
https://github.com/alibaba/sealer/runs/2938256523?check_suite_focus=true
### Special notes for reviews
link to issue #353

### CODE REVIEW

File: common/common.go [hunk 1]
```diff
@@ -101,7 +101,7 @@ const (
 	CdAndExecCmd    = "cd %s && %s"
 	TagImageCmd     = "%s tag %s %s"
 	PushImageCmd    = "%s push %s"
-	BuildClusterCmd = "%s build -f %s -t %s -b %s ."
+	BuildClusterCmd = "%s build -f %s -t %s -c %s -b %s"
```
Comment (fanux) [inline common/common.go:104]: I think we don't need add -c? fllow Docker build command line?

Comment (kakaZhou719) [reply common/common.go:104]: if user set context dir ,we need to set it . when build context is not set, we can use  "."  by default.

Comment (fanux) [reply common/common.go:104]: Why not `sealer build -t xxx .` but `sealer build -t xxx -c .`?  We need context but needn't -c flag I think

File: sealer/cmd/build.go [hunk 2]
```diff
@@ -25,18 +25,21 @@ import (
...
 
 // buildCmd represents the build command
 var buildCmd = &cobra.Command{
-	Use:     "build",
+	Use:     "build [flags] PATH",
 	Short:   "cloud image local build command line",
 	Example: `sealer build -f Kubefile -t my-kubernetes:1.18.3 .`,
 	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 0 {
+			logger.Error("sealer build requires exactly 1 argument.")
```
Comment (bxy4543) [inline sealer/cmd/build.go:40]: if args ==0 ，context = "."  ，  else context = arg0
                   i think

**Review by fanux — APPROVED** [review]

**Pull request merged**